src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -873,10 +911,16 @@ export default class DataTable extends Vue {
       if (columnName && sortIcon) {
         sortIcon.className = `${ICON_CLASS} -xs ${DATA_TABLE_SORT_ICONS.ARROW}`;
         if (currentSort === 'ascending' && this._sortConfig && this._sortConfig.key === columnName) {
+          const sortingData: DataTableSorting = {
+            column: columnName,
+            direction: 'descending',
```

Comment:
Shouldn't this be ascending?

Reply:
Nope, in this scope `currentSort === 'ascending'` means that the user has already applied ascending sorting to the column and the next one should switch direction.

Here is a demo:
https://user-images.githubusercontent.com/25136131/123399730-417d8f80-d5a5-11eb-8e55-a98b12495324.mov

Reply:
Got it, thanks!